client/components/Sidebar.vue

```diff
@@ -2,17 +2,16 @@
 	<aside id="sidebar" ref="sidebar">
 		<div class="scrollable-area">
 			<div class="logo-container">
+				<h1 class="sr-only">The Lounge</h1>
```

Comment:
My impression is that before the `alt` attributing was detailing the image logo, while moving it to an `<h1>` heading probably changes its meaning to be the service's name. So it should maybe be moved outside of the `<div class="logo-container">` and maybe even outside of  `<aside id="sidebar" ref="sidebar">`

Another problem is that on the connect page there is already an `<h1>` heading.
Example of the one used on [the demo instance](https://demo.thelounge.chat/#/connect):
`<h1 class="title">Connect to Libera.Chat</h1>`

Also it would be useful if that service name could be configurable in particular for people that might be connected to multiple The Lounge instances to more easily differentiate them.
I suppose that last comment might actually best addressed in a separate issue and PR as it also affect the html page's title which is currently set to `<title>The Lounge</title>`.

Reply:
> Another problem is that on the connect page there is already an `<h1>` heading.

The edit network page also uses an `<h1>` for its heading.

Reply:
Oof, thanks for catching this; I hadn't realized that some of the forms have h1's on them. I'm kind of inclined to just back out my logo/h1 change for now.

However, this may still be problematic on a further level, because the h1 present on various forms also appears later in the document than the h2 I added for Network and Channel List, which seems not ideal.

It seems as if the forms might be the only places where headings already existed prior to this PR - almost as if headings were _only_ considered for the forms and not for the rest of the UI that is in use the majority of the time. This may make adding headings to the rest of the UI less straightforward unless they were to be re-leveled within forms?

Reply:
I don't see a huge problem with multiple `<h1>`s. I know there are some who would say this is a "bad thing" but I am not one of those. Ultimately this is about exposing the structure of the page to make it easier for assistive technology users to navigate around.

Reply:
I had thought that having a double `<h1>` could bring accessibility issues, but if it doesn't then I'm fine with this part of the changes.